docs/processing.md

```diff
@@ -208,15 +239,24 @@ exporters:
   otlp:
 
 processors:
-  query:
+  transform:
     # Assuming group_by is defined in a contrib extension module, not baked into the "query" processor
     extensions: [group_by]
-    expressions:
-      - drop() where attributes["http.target"] = "/health"
-      - delete(attributes["http.request.header.authorization"])
-      - replace_wildcards("/user/*/list/*", "/user/{userId}/list/{listId}", attributes["http.target"])
-      - set(attributes["k8s_pod"], resource.attributes["k8s.pod.name"]) from metric
-      - group_by(trace_id, 2m) from span
+    traces:
+      queries:
```

Comment:
so only for "traces" you have "queries"?

Reply:
Good catch, they should all have "queries"